src/main/java/com/redhat/coolstore/rest/ProductEndpoint.java

```diff
@@ -3,18 +3,16 @@
 import java.io.Serializable;
 import java.util.List;
 
-import javax.enterprise.context.RequestScoped;
-import javax.inject.Inject;
-import javax.ws.rs.*;
-import javax.ws.rs.core.MediaType;
+import jakarta.enterprise.context.RequestScoped;
+import jakarta.inject.Inject;
+import jakarta.ws.rs.*;
+import jakarta.ws.rs.core.MediaType;
 
 import com.redhat.coolstore.model.Product;
 import com.redhat.coolstore.service.ProductService;
 
 @RequestScoped
 @Path("/products")
-@Consumes(MediaType.APPLICATION_JSON)
```

Comment:
Should this be removed?

Reply:
Yes it can be removed since both the methods are GET calls in this end point. However there should be an @Produces(MediaType.APPLICATION_JSON)